app/routes/pages/components/PageHierarchy.tsx

```diff
@@ -85,15 +85,22 @@ type AccordionProps = {
   currentCategory: string;
 };
 
-const AccordionContainer = ({ title, children }: AccordionProps) => {
-  const location = useLocation();
-  const currentCategory = location.pathname.split('/')[2];
+const AccordionContainer = ({
+  title,
+  children,
+  currentCategory,
```

Comment:
lol did it have access to this all along 💀

Reply:
Heheh it was even typed already(: so why it wasn't used is quite the mystery to me